### PR TITLE
SAK-30388: navigation tabs in assignments is not always showing the correct selected tab

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/SAK-30388

I took the approach of just using a dedicated 'view' for the grade report UI.

@ern, what say you, good sir?

### CODE REVIEW

File: assignment/assignment-tool/tool/src/webapp/vm/assignment/assignment_macros.vm [hunk 1]
```diff
@@ -117,7 +117,7 @@
 
         #if( $withGrade && $!allowGradeSubmission )
             <li #if( $prevAction == false ) class="firstToolBarItem" #set( $prevAction = true ) #end>
-                #if( !$!view.equals( "all" ) )
```
Comment (ern) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/assignment_macros.vm]: Should view "all" not be linked with any tab then?

Comment (bjones86) [inline assignment/assignment-tool/tool/src/webapp/vm/assignment/assignment_macros.vm]: Nope, it's never had an associated tab.

**Comment by ern** [comment]
LGTM

**Comment by bjones86** [comment]
Thanks @ern !!

**Pull request merged**